readme.rst

```diff
@@ -43,91 +30,57 @@ Try ``warcprox --help`` for documentation on command line options.
-Since the bytes are encrypted, the proxy can't make sense of the information
-it's proxying. This nonsensical encrypted data would not be very useful to
-archive.
-
-In order to capture https traffic, warcprox acts as a "man-in-the-middle"
-(MITM). When it receives a ``CONNECT`` directive from a client, it generates a
-public key certificate for the requested site, presents to the client, and
-proceeds to establish an encrypted connection with the client. Then it makes a
-separate, normal https connection to the remote site. It decrypts, archives,
-and re-encrypts traffic in both directions.
-
-Although "man-in-the-middle" is often paired with "attack", there is nothing
-malicious about what warcprox is doing. If you configure an instance of
-warcprox as your browser's http proxy, you will see lots of certificate
-warnings, since none of the certificates will be signed by trusted authorities.
-To use warcprox effectively the client needs to disable certificate
-verification, or add the CA cert generated by warcprox as a trusted authority.
-(If you do this in your browser, make sure you undo it when you're done using
-warcprox!)
+Normally, HTTP proxies can't read encrypted HTTPS traffic. The browser uses the HTTP ``CONNECT`` method to establish a tunnel through the proxy, and the proxy merely routes raw bytes between the client and server. Since the bytes are encrypted, the proxy can't make sense of the information that it proxies. This nonsensical encrypted data is not typically useful for web archiving purposes.
+
+In order to capture HTTPS traffic, warcprox acts as a "man-in-the-middle" (MITM). When it receives a ``CONNECT`` directive from a client, it generates a public key certificate for the requested site, presents to the client, and proceeds to establish an encrypted connection with the client. It then makes a separate, normal HTTPS connection to the remote site. It decrypts, archives, and re-encrypts traffic in both directions.
+
+Configuring a warcprox instance as a browser’s HTTP proxy will result in security certificate warnings because none of the certificates will be signed by trusted authorities. However, there is nothing malicious about warcprox functions. To use warcprox effectively, the client needs to disable certificate verification or add the CA certification generated by warcprox as a trusted authority. When using the latter, remember to undo this change when finished using warcprox.
 
 API
 ===
-For interacting with a running instance of warcprox.
+The warcprox API may be used to retrieve information from and interact with a running warcprox instance, including:
 
-* ``/status`` url
-* ``WARCPROX_WRITE_RECORD`` http method
-* ``Warcprox-Meta`` http request header and response header
+* Retrieving status information via ``/status`` URL
+* Writing WARC records via ``WARCPROX_WRITE_RECORD`` HTTP method
+* Querying and editing ``Warcprox-Meta`` HTTP request header and response header
```

Comment:
"Querying and editing" doesn't totally make sense to me. Maybe something like "Controlling some warcprox settings per capture via the ``Warcprox-Meta`` HTTP request header". Not sure how to cover the response header in the same sentence fragment concisely and accurately, but I think it's ok to leave it out here.